### PR TITLE
Molo profiles upgrade 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@codiebeulaine @projectmushroom please review 

screen shot 2017-06-14 at 3 40 58 pm
screen shot 2017-06-14 at 3 42 20 pm
screen shot 2017-06-14 at 3 42 33 pm
screen shot 2017-06-14 at 3 43 07 pm
screen shot 2017-06-14 at 3 43 37 pm

File: tuneme/templates/new/profiles/done.html [hunk 1]
```diff
@@ -3,67 +3,70 @@
...
         {{ form.location }}
-        <span class="input-hint">{% trans "Only you will see this" %}</span>
+        <p class="helptext">{% trans "Only you will see this" %}</p>
       </fieldset>
       {% endif %}
 
       {% if profile_settings.activate_education_level and not profile_settings.capture_education_level_on_reg %}
       <fieldset>
-        <label for="education_level">{% trans "WHAT IS YOUR HIGHEST LEVEL OF EDUCATION?" %}</label><br>
+        <label for="education_level">{% trans "What is your highest level oof education?" %}</label>
```
Comment (codiebeulaine) [inline tuneme/templates/new/profiles/done.html]: spelling mistake

File: tuneme/templates/new/profiles/done.html [hunk 2]
```diff
@@ -3,67 +3,70 @@
...
       <fieldset>
-        <label for="alias">{% trans "DISPLAY NAME" %}</label><br>
-          {{ form.alias.errors }}
-          {{ form.alias }}
-          <span class="input-hint">{% trans "We value your online security, therefore we hide your username and allow you to comment and interact with the site with a display name." %}</span>
+        <label for="alias">{% trans "Display Name" %}</label>
+        {{ form.alias.errors }}
+        {{ form.alias }}
+        <p class="helptext">{% trans "We value your online security, therefore we hide your username and allow you to comment and interact with the site anonymously." %}</p>
+        <p class="helptext">{% trans "If you would like to interact with other people, like your friends to be able to see you as a username and not anonymous, update your username here or select to keep your username as your display name." %}</p>
```
Comment (codiebeulaine) [inline tuneme/templates/new/profiles/done.html]: should have another comma after friends

**Comment by codiebeulaine** [comment]
two comments but 👍

**Pull request merged**